cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -529,6 +529,24 @@ rules:
       - "*"
     verbs:
       - "*"
+  - apiGroups:
```

Comment:
When the RBAC changes in common.yaml, we also need to update the helm chart [templates](https://github.com/rook/rook/tree/master/cluster/charts/rook-ceph/templates).